### ISSUE TITLE
Merge `feature/layerpicker` into `develop`

### ISSUE THREAD

**Issue by arottersman** [WikiWatershed/model-my-watershed#1728]

**Comment by ajrobbins** [comment]
Waiting on this until after 3/9/17 to allow Staging to serve as a fallback to prod for EPA demo.

**Comment by arottersman** [comment]
Closing: duplicate of #1737
